tests/test_container_build.py

```diff
@@ -35,10 +35,14 @@
     url=OPENSUSE_BUSYBOX_URL,
     custom_entry_point="/bin/sh",
 )
-#: This is just a busybox container with 4MB of random data in there
+
+#: This is just a busybox container with 4MB of random data
+#: and a custom entrypoint
 BUSYBOX_WITH_GARBAGE = DerivedContainer(
     base=BUSYBOX_WITH_ENTRYPOINT,
-    containerfile="""RUN dd if=/dev/random of=/foobar bs=4M count=1
+    containerfile="""
+ENTRYPOINT /usr/bin/id
```

Comment:
Why did you add the `ENTRYPOINT` here?

Reply:
I've now splitted it into a separate test, which hopefully helps.